docs/operations/storageclasses.md

```diff
@@ -122,3 +122,44 @@ parameters:
   csi.storage.k8s.io/secret-name: storageos-api
   csi.storage.k8s.io/secret-namespace: storageos
 ```
+
+### Topology volume Storage Class
+
+StorageClass that utilizes kubernetes node topology zones and a list of
+preferred and required zones to decide where its deployments should live.
+
+More on [k8s storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes/#allowed-topologies)
+and csi specs for [create volume](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume)
```

Comment:
I think the csi spec might be too much info for our customers. 

I'm happy to defer to other people's opinions though

Reply:
I will update to only include the first one